nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml

```diff
@@ -0,0 +1,13 @@
+bins:
+- k1:
+    min: null
+    mid: 1.0e-09
+    max: null
+  k2:
+    min: null
+    mid: 2.7225
+    max: null
+  k3:
+    min: null
+    mid: 0.0
+    max: null
```

Comment:
I think they should all read from the same kinematics file given that's the same.

Also, add the right variables :__

Reply:
To both points: I just followed the implementation of the already existing ones, but I suppose I can fix all

Reply:
If you fix them all even better.

Btw, if you change the processes from `DIS_whatever` to `DIS_INTEG_whatever` I'd appreciate it.

Reply:
https://github.com/NNPDF/nnpdf/pull/2059/commits/928a27f5f80e7ae4c96f298823606e4bf5434886

Just `INTEG_` I guess, or do you want to ask @comane to change it?

Reply:
Ah, yes. Even better.